w_common/lib/src/browser/disposable_browser.dart

```diff
@@ -243,16 +243,17 @@ class Disposable implements disposable_common.Disposable {
           onError: onError, onDone: onDone, cancelOnError: cancelOnError);
 
   @override
-  T manageAndReturnTypedDisposable<T extends disposable_common.Disposable>(
-          T disposable) =>
-      _disposable.manageAndReturnTypedDisposable(disposable);
+  T manageAndReturnTypedDisposable<T extends disposable_common.Disposable?>(
+      T disposable) {
```

Comment:
This is what I had originally, but in a group code review with FEDX, we landed on the current form because we didn't want to require that consumers do this:
```
var foo = manageAndReturnTypedDisposable(Foo());
foo?.bar();
```
We want consumers to be able to do:
```
var foo = manageAndReturnTypedDisposable(Foo());
foo.bar();
```

Reply:
Oh interesting! So they are not equivalent. Nice, I like the ergonomics of that better. Let's keep it as you have it.